apps/cowswap-frontend/src/legacy/state/orders/utils.ts

```diff
@@ -51,9 +51,7 @@ export function isOrderFulfilled(
  */
 export function isOrderCancelled(order: Pick<EnrichedOrder, 'creationDate' | 'invalidated' | 'status'>): boolean {
   const creationTime = new Date(order.creationDate).getTime()
-  return (
-    (order.invalidated && Date.now() - creationTime > PENDING_ORDERS_BUFFER) || order.status === OrderStatus.CANCELLED
-  )
+  return order.invalidated && Date.now() - creationTime > PENDING_ORDERS_BUFFER
```

Comment:
This was a part of the fix: https://github.com/cowprotocol/cowswap/pull/2914

Wouldn't it brake it?

Reply:
Well, to be fair, it's broken already.

This is on barn:
![image](https://github.com/cowprotocol/cowswap/assets/43217/b03dd90b-c387-46b4-8d9f-33e62e5b12a9)

But I'll isolate the behaviour just to not block this fix